CollectionViewTools.xcodeproj/project.pbxproj

```diff
@@ -477,7 +478,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.rosberry.CollectionViewTools;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
-				SWIFT_VERSION = 4.2;
+				SWIFT_VERSION = 5.0;
```

Comment:
Are there any side effects of moving from swift 4.2 to swift 5?

Reply:
I think no